src/lsm/manifest.zig

```diff
@@ -222,11 +225,46 @@ pub fn ManifestType(comptime Table: type, comptime Storage: type) type {
             manifest_level.set_snapshot_max(snapshot, table);
             assert(table.snapshot_max == snapshot);
 
-            // Append update changes to the manifest log
+            // Append update changes to the manifest log.
             const log_level = @intCast(u7, level);
             manifest.manifest_log.insert(log_level, table);
         }
 
+        pub fn move_table(
+            manifest: *Manifest,
+            level_a: u8,
+            level_b: u8,
+            snapshot: u64,
+            table: *TableInfo,
+        ) void {
+            assert(level_b == level_a + 1);
+            assert(level_b < constants.lsm_levels);
+            assert(table.snapshot_max >= snapshot);
+
+            const manifest_level_a = &manifest.levels[level_a];
+            const manifest_level_b = &manifest.levels[level_b];
+            if (constants.verify) {
+                assert(manifest_level_a.contains(table));
+                assert(!manifest_level_b.contains(table));
+            }
+
+            // Insert the table into level b and append insert changes to the manifest log.
```

Comment:
```suggestion
            // Insert the table into level A and append insert changes to the manifest log.
```

Same in the comments below, we can capitalize `A` or `B`.